Tasks/AzureFunctionAppV1/package-lock.json

```diff
@@ -70,6 +70,7 @@
         "decompress-zip": "0.3.0",
         "jsonwebtoken": "7.3.0",
         "ltx": "2.6.2",
+        "node-stream-zip": "1.7.0",
```

Comment:
this version is OSS verified but not IP scanned. 
Requested for IP scan and also there is no other version that is IP scanned.

Reply:
the above component is OSS approved and IP scanned. :)
Waiting for TPN.